### ISSUE TITLE
node trying to resolve DNS name: db

### ISSUE THREAD

**Issue by pysiak** [mynodebtc/mynode#445]
**Describe the bug**
I'm observing repetetive attempts by mynodebtc to resolve a domain: db
What's that? Is that indicative of a bug or misconfig? what piece of the setup is causing it?
Note, I changed my DNS to be my local pihole, so the screenshot is showing local resolution attempts.
However, this was happening before I switched it.

**Expected behavior**
I'd expect mynodebtc to look for DNS names that we've got configured.

**Screenshots**
![image](https://user-images.githubusercontent.com/2396137/101260939-a53c8a00-3733-11eb-8a48-96b39e5d2e51.png)

**Desktop (please complete the following information):**

**MyNode hardware (please complete the following information):**
 - Device: raspi4
 - Version 0.2.19

**Additional context**

**Comment by Jacksper13** [comment]
@pysiak is this still a thing on mynode v0.2.30 or v0.2.31?

**Comment by oitdmser** [comment]
I'm seeing this as of version 0.3.03

**Comment by tehelsper** [comment]
The "db" hostname is one of the Dojo containers. Have you changed anything with Dojo recently? Is it running properly?

**Comment by oitdmser** [comment]
I've never run Dojo on this node and it's currently disabled.

**Comment by oitdmser** [comment]
Update: I enabled and re-disabled Dojo and the queries have seemed to have stopped.

**Comment by oitdmser** [comment]
These queries began again after upgrading to 0.3.04. Upgraded to 0.3.05 and the query storm continued. Uninstalled Dojo and the problem resolves. There still seems to be an issue here.

**Comment by tehelsper** [comment]
Was Dojo installed but disabled when it started querying for db again after the upgrade?

**Comment by oitdmser** [comment]
Correct. Dojo was installed but disabled before I upgraded. After the upgrade it was still disabled, but the db query storm began and continued until I uninstalled Dojo completely.

**Comment by tehelsper** [comment]
Interesting. If it happens again can you run:

docker ps
sudo systemctl status dojo

It seems part of Dojo must be getting started again after an upgrade somehow. I'll see if I can reproduce it.
